### PR TITLE
Upgrade swiss-ng-api.yaml to the NextGenPSD2 XS2A Framework version 1.3.8_2020-12-14

### PR DESCRIPTION
**Pull request by wishopt**
Upgrade the swiss-ng-api.yaml to reflect the changes made between versions 1.3.4_2019-07-17v1 and 1.3.8_2020-12-14 of the Berling Group NextGenPSD2 Access to Account Interoperability Framework.

### CODE REVIEW

File: swiss-ng-api.yaml
Comment (yves-bussard) [inline swiss-ng-api.yaml]: This endpoint has not disappeared with BG 1.3.8. You should not delete it.

**Review by yves-bussard — CHANGES_REQUESTED** [review]

File: swiss-ng-api.yaml
Comment (wishopt) [reply swiss-ng-api.yaml]: Hello Yves
The endpoint is restored again. I also checked the changes to the maxLength attributes, as asked in the last meeting. They're all changes made during the update, as far as I can tell. Thanks for the feedback.

**Comment by yves-bussard** [comment]
I cannot approve this PR. Probably, because you never invited me as a reviewer. While you're at it, please also invide @chatelao as a reviewer.

**Comment by openbankingproject** [comment]
@yves-bussard you should have write access now :-)

**Review by yves-bussard — APPROVED** [review]

**Pull request merged**
